src/app/healthyrelationships/healthyrelationships.page.html

```diff
@@ -23,137 +23,134 @@
 
   <div *ngIf="subTab==='healthy'">
     <h1 style="font-size: 18px;">
-      Healthy, whether it's friendship or romantic
+      Signs of a Healthy Relationship
     </h1>
+    <h3 style="text-align: center; font-size: 14px;">
+      (friendship or romantic)
+    </h3>
     <ion-slides pager="true" [options]="slideOpts">
       <ion-slide>
-        <ion-card>
-          <img
-            src="https://static1.squarespace.com/static/58ebeae859cc68b20f410b26/5a281fa5e2c4838eb6e38450/5a302bb5419202a08e5e0347/1513106439847/listen.jpg?format=1500w"
-          />
+        <ion-card class="slide-card">
```

Comment:
I added this slide-card class to make the card height consistent. I also resized all the images just for the "healthy" sub-tab. So, now you will notice when you scroll through the healthy sub-tab, everything is the same exact height.